maven/tests/run.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: maven-run
+spec:
+  inputs:
+    resources:
+    - name: source
+      resourceRef:
+        name: maven-resource-petclinic
+#    params:
+#    - name: MAVEN_MIRROR_URL
```

Comment:
can you run a http mirror as sidecar?

Reply:
let me try that

Reply:
nm, i don't think this would work since the mirrors are ephemeral, it would be good to see with the other plumbing folks if we can setup a permanent mirror in there to speed up CI tasks,

Reply:
right, so then i shall write the test run.yaml without a MAVEN_MIRROE_URL param